### PR TITLE
Add gift icon

### PR DESCRIPTION
**Pull request by usame-algan**
Adding a new icon needed for [#3413](https://github.com/gnosis/safe-react/issues/3413)

## Screenshot
Screenshot 2022-02-08 at 13 43 39

### CODE REVIEW

**Review by iamacook — APPROVED** [review]
👍

File: src/dataDisplay/Icon/images/gift.tsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+
+export default {
+  sm: (
+    <svg
```
Comment (katspaugh) [inline src/dataDisplay/Icon/images/gift.tsx:5]: Did you run it thru svgo?

Comment (katspaugh) [reply src/dataDisplay/Icon/images/gift.tsx:5]: https://github.com/svg/svgo

**Review by katspaugh — APPROVED** [review]
👍

**Pull request merged**